app/assets/v2/js/grants/new.js

```diff
@@ -69,6 +69,7 @@ $(document).ready(function() {
 
       web3.eth.getAccounts(function(err, accounts) {
         web3.eth.net.getId(function(err, network) {
+          network = 'testrpc';
```

Comment:
that is a hangover from my still using testrpc. I get an error when using the network from browser because it is a 13 digit number and the model is only 8. I will redact it.